components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -17,8 +17,7 @@ It helps you monitor activities and usage of your Keboola Connection projects. I
 To configure the extractor, select one of the following modes:
  
 1. **Project mode** --- extracts data from a selected Keboola Connection project only.
-2. **Organization mode** --- extracts data from all projects of your organization. The data is extracted 
-to a single target project that must be set via your Keboola Customer Success Manager.
+2. **Organization mode** --- extracts data from all projects of your organization. The data is extracted to a single target project. This must be set up by Keboola: Configure the extractor and contact your Keboola Customer Success Manager or our support. Provide the ID of the project containing the extractor configuration.
```

Comment:
link na contact keboola (asi https://help.keboola.com/management/support/)

zase je tam to "Provide the ID of the project containing the extractor configuration." to se ale dělá samo, pokud to jde z projektu, kde se to furt bere tohle?

Reply:
:-)) 
Tentokrat je zdroj David Esner a Matejkys to posvetil. Davidovi nebylo dostatecne jasny ... must be set via your CSM..., tak to prepsal a poznamku o ID doplnil. Upravila jsem to trochu. Mela jsem puvodne pridat i to, kde lidi ID projektu najdou, ale odolala jsem :).  

Takze to vyhodim a doplnim link na support. (A uz si to budu pamatovat! Jen je divny, ze lidi, co delaji se zakaznikama a maji jim radit, tak tohle nevi.) 
Dik!!